linux_os/guide/system/software/sudo/sudo_restrict_privilege_elevation_to_authorized/rule.yml

```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+title: 'The operating system must restrict privilege elevation to authorized personnel'
+
+prodtype: sle15
+
+description: |-
+    The sudo command allows a user to execute programs with elevated
+    (administrator) privileges. It prompts the user for their password
+    and confirms your request to execute a command by checking a file,
+    called sudoers.
+
+rationale: |-
+    If the "sudoers" file is not configured correctly, any user defined
+    on the system can initiate privileged actions on the target system.
+
+severity: medium
+
+identifiers:
+    cce@sle15: CCE-85712-8
+
+references:
+    nist: CM-6(b),CM-6(iv)
+    disa@sle15: CCI-000366
+    stig@sle15: SLES-15-020101
+
+ocil_clause: 'Verify the "sudoers" file restricts sudo access to authorized personnel.'
+
+ocil: |-
+    To determine if "sudoers" file, restricts sudo access, run the following commands:
+    <pre>$ sudo grep -PR '^\s*ALL\s+ALL\=\(ALL\)\s+ALL\s*$' /etc/sudoers /etc/sudoers.d/*</pre>
+    <pre>$ sudo grep -PR '^\s*ALL\s+ALL\=\(ALL\:ALL\)\s+ALL\s*$' /etc/sudoers /etc/sudoers.d/*</pre>
+    Both commands should return no output.
+
+platform: sudo
+
+warnings:
+  - general: This rule doesn't come with a remediation, as the exact requirement allows exceptions, and removing lines from the sudoers file can make the system non-administrable.
```

Comment:
can you break this text into multiple lines?

Reply:
Sure